internal/gengapic/gengapic.go

```diff
@@ -153,6 +138,37 @@ func (g *generator) gen(serv *descriptor.ServiceDescriptorProto) error {
 		}
 	}
 
+	return nil
+}
+
+// gen generates client for the given service.
+func (g *generator) gen(serv *descriptor.ServiceDescriptorProto) error {
+	servName := pbinfo.ReduceServName(*serv.Name, g.opts.pkgName)
```

Comment:
Done

Reply:
It's just tricky...the fields are exported for setting, but should be accessed by the accessor in order to allow protobuf unwrapping logic. For example, if a `*string` is unset/`nil`, `GetName()` gives you `""` instead of nil.